### DISCUSSION TITLE
MeshCentral security.

### DISCUSSION THREAD

**Discussion by artem33region** [Ylianst/MeshCentral#4487] [Q&A]
Hello!
Where can i get best practices of implement MeshCentral in Hybrid mode?
Can i implement MeshCentral how Front-end and Back-end?

**Comment by Ylianst** [comment]
Hi. I am not sure I understand your question and/or these are rather broad questions. I have a [video on LAN/WAN and Hybrid mode here](https://www.youtube.com/watch?v=gx5Fh3pQOns). MeshCentral is a single application, but you can put a reverse proxy in front of it and you have you pick of databases that are supported (MongoDB, MySQL, MariaDB, PostgesSQL) for the back-end.

**Reply by renueman** [reply]
I think he is asking about "Best Practices" for security using meshcentral.
